lib/prop/limiter.rb

```diff
@@ -157,6 +161,8 @@ def _throttle(strategy, handle, key, cache_key, options)
           strategy.decrement(cache_key, options.fetch(:decrement), options) :
           strategy.increment(cache_key, options.fetch(:increment, 1), options)
 
+        after_evaluated_callback.call(handle, counter, options) if after_evaluated_callback
```

Comment:
Oh actually one thing - it looks like this won't work for the leaky bucket strategy, because that calls into a different implementation further up.

You could be lazy and call the callback from in there, but the right thing to do is see if we can refactor this to know about the leaky-bucket counter remaining in this method.